cs-office-365/src/main/java/io/cloudslang/content/office365/utils/Descriptions.java

```diff
@@ -96,4 +96,11 @@
         public static final String SUCCESS_DESC = "Token generated successfully.";
         public static final String FAILURE_DESC = "There was an error while trying to retrieve token.";
     }
+
+    public static class ListMessages {
+        public static final String RETURN_RESULT_DESC = "If successful, returns the message bodies.";
```

Comment:
I would use here something like "returns the complete API response containing the messages."